modules/dxp/apps/search-experiences/search-experiences-test/src/testIntegration/java/com/liferay/search/experiences/service/test/SXPElementLocalServiceTest.java

```diff
@@ -172,6 +175,12 @@ private SXPElement _addSXPElement(String externalReferenceCode, long userId)
 		return sxpElement;
 	}
 
+	@DeleteAfterTestRun
+	private Company _company;
+
+	@Inject
+	private CompanyLocalService _companyLocalService;
```

Comment:
does CompanyLocalService need to be here or is it left over from the original changes?

Reply:
lefted over